nexus/src/authz/omicron.polar

```diff
@@ -180,7 +199,9 @@ resource Organization {
 	"modify" if "admin";
 
 	relations = { parent_silo: Silo };
-	"admin" if "admin" on "parent_silo";
+	"admin" if "collaborator" on "parent_silo";
+	"read" if "viewer" on "parent_silo";
```

Comment:
I found that Silo Viewers couldn't actually see inside an Organization.

We probably want to break down and add an "Organization Viewer" role.  It wasn't obviously necessary when I was writing out the initial roles.  But at this point, it's weirder _not_ to have it.  Still, that's a bigger change so for now I'm just granting these two permissions directly.